pom.xml

```diff
@@ -51,12 +51,13 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <workflow.version>1.4</workflow.version>
-        <jenkins.version>2.7</jenkins.version>
+        <jenkins.version>2.230</jenkins.version>
```

Comment:
This means that the plugin won't be compatible with older Jenkins (namely no current LTS will be able to use it). Probably you'd rather support several LTS.

I personnaly would choose something like 2.190 or 2.176 but I don't have a lot of experience to choose this others might have better advices. You can also have a look to  https://stats.jenkins.io/pluginversions/ghprb.html to decide.